test/dynamo/test_dynamic_shapes.py

```diff
@@ -85,10 +85,12 @@ def make_dynamic_cls(cls):
         DynamicShapesReproTests.test_dynamic_shapes_float_guard_dynamic_shapes  # noqa: F821
     )
 
-    # TODO model is somehow not being freed when z3 is available
-    unittest.expectedFailure(
-        DynamicShapesMiscTests.test_parameter_free_dynamic_shapes  # noqa: F821
-    )
+
+    if not config.inline_inbuilt_nn_modules:
+        # TODO model is somehow not being freed when z3 is available
+        unittest.expectedFailure(
```

Comment:
Can you file an issue for this? this sounds very strange

Reply:
https://github.com/pytorch/pytorch/issues/127444